braviarc/braviarc.py

```diff
@@ -98,7 +98,7 @@ def _wakeonlan(self):
 
     def send_req_ircc(self, params, log_errors=True):
         """Send an IRCC command via HTTP to Sony Bravia."""
-        headers = {'SOAPACTION': 'urn:schemas-sony-com:service:IRCC:1#X_SendIRCC'}
+        headers = {'SOAPACTION': '"urn:schemas-sony-com:service:IRCC:1#X_SendIRCC"'}
```

Comment:
When I was looking for the command using this link http://openremote.org/display/forums/Sony+TV+HTTP+control it looked like the command is being used being used as the mute command. But I found somewhere else that this same command actually switches on the tv.
I have tested it using X80C and X85 and it is working on both